.azuredevops/build-and-push-dev.yml

```diff
@@ -54,11 +54,12 @@ stages:
               arguments: |
                 --pull
                 --cache-from $(azureContainerRegistry.domain)/$(azureContainerRegistry.repository):latest
+                --build-arg home=/app
                 --build-arg NEXT_BUILD_DATE=$(app.buildDate)
                 --build-arg NEXTAUTH_URL=$(app.domain)
                 --build-arg NEXT_AUTH_USERNAME=$(NEXT_AUTH_USERCODE)
```

Comment:
NEXT-AUTH-USERCODE and NEXT-AUTH-PASSWORD
NEXT_AUTH_SECRET the variable does not have a value